usermanager/src/test/java/org/pesmypetcare/usermanager/clients/pet/PetCollectionsManagerClientTest.java

```diff
@@ -0,0 +1,644 @@
+                + "    \"key\":\"1990-01-08T15:20:30\"\n"
+                + "  },{\"body\":{\n"
+                + "      \"name\":\"Planking\",\n"
+                + "      \"coordinates\":[\n"
+                + "        {\n"
+                + "          \"latitude\":84,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":34,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":21,\n"
+                + "          \"longitude\":58\n"
+                + "        }\n"
+                + "      ],\n"
+                + "      \"description\":\"Staying horizontal over a plain surface\",\n"
+                + "      \"endDateTime\":\"2000-01-08T15:20:30\"\n"
+                + "    },\n"
+                + "    \"key\":\"1995-01-08T15:20:30\"\n"
+                + "  },{\"body\":{\n"
+                + "      \"name\":\"Planking\",\n"
+                + "      \"coordinates\":[\n"
+                + "        {\n"
+                + "          \"latitude\":84,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":34,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":21,\n"
+                + "          \"longitude\":58\n"
+                + "        }\n"
+                + "      ],\n"
+                + "      \"description\":\"Staying horizontal over a plain surface\",\n"
+                + "      \"endDateTime\":\"2000-01-08T15:20:30\"\n"
+                + "    },\n"
+                + "    \"key\":\"1998-01-08T15:20:30\"\n"
+                + "  }\n"
+                + "]");
+        exerciseDataJson = new StringBuilder("{\n"
+                + "      \"name\":\"Planking\",\n"
+                + "      \"coordinates\":[\n"
+                + "        {\n"
+                + "          \"latitude\":84,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":34,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":21,\n"
+                + "          \"longitude\":58\n"
+                + "        }\n"
+                + "      ],\n"
+                + "      \"description\":\"Staying horizontal over a plain surface\",\n"
+                + "      \"endDateTime\":\"2000-01-08T15:20:30\"\n"
+                + "}"
+        );
+        List<LatLng> coordinates = new ArrayList<>();
+        coordinates.add(new LatLng(84, 58));
+        coordinates.add(new LatLng(34, 58));
+        coordinates.add(new LatLng(21, 58));
+        exerciseData = new ExerciseData("Planking", "Staying horizontal over a plain surface",
+                "2000-01-08T15:20:30", coordinates);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "2000-01-08T15:20:30" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5549434)